core/src/main/kotlin/DokkaBootstrapImpl.kt

```diff
@@ -11,12 +11,26 @@ import java.util.function.BiConsumer
 fun parsePerPackageOptions(args: List<String>): List<PackageOptions> = args.map { it.split(",") }.map {
     val prefix = it.first()
     if (prefix == "")
-        throw IllegalArgumentException("Please do not register packageOptions with all match pattern, use global settings instead")
+        throw IllegalArgumentException(
+            "Please do not register packageOptions with all match pattern, use global settings instead"
+        )
+
     val args = it.subList(1, it.size)
-    val deprecated = args.find { it.endsWith("deprecated") }?.startsWith("+") ?: true
-    val reportUndocumented = args.find { it.endsWith("reportUndocumented") }?.startsWith("+") ?: true
-    val privateApi = args.find { it.endsWith("privateApi") }?.startsWith("+") ?: false
-    val suppress = args.find { it.endsWith("suppress") }?.startsWith("+") ?: false
+
+    val deprecated = args.find { it.endsWith("deprecated") }?.startsWith("+")
+        ?: args.find { it.endsWith("skipDeprecated") }?.startsWith("+")
```

Comment:
Since we're changing the configuration quite extensively I think we can just drop the old argument names like "deprecated" and "privateApi"

Reply:
✅ Removed support for those "old" argument names with additional commit